modules/library/coverage/src/main/java/org/geotools/image/ImageWorker.java

```diff
@@ -386,6 +436,19 @@ public ImageWorker(final File input) throws IOException {
      */
     public ImageWorker(final RenderedImage image) {
         inheritanceStopPoint = this.image = image;
```

Comment:
Wouldn't it make sense to call setImage here? Or do we have different semantics in that method and this one?